spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -862,7 +862,7 @@ class CometSparkSessionExtensions
               newOp match {
                 case Some(nativeOp) =>
                   s.child match {
-                    case n if n.isInstanceOf[CometNativeExec] || !n.supportsColumnar =>
+                    case n if n.isInstanceOf[CometNativeExec] || n.supportsColumnar =>
```

Comment:
If the child is not a `CometNativeExec` can be be a `ShuffleExchangeExec`? In that case we should not use `CometColumnarShuffle`.

Reply:
I see, thank you. CometShuffleExchangeExec supports row base input.